x-pack/plugins/security/server/lib/authentication/providers/token.test.ts

```diff
@@ -183,83 +193,83 @@ describe('TokenAuthenticationProvider', () => {
       const user = { username: 'user' };
 
       // GetUser will be called with request's `authorization` header.
-      callWithRequest.withArgs(request, 'shield.authenticate').returns(Promise.resolve(user));
+      callWithRequest.withArgs(request, 'shield.authenticate').resolves(user);
 
-      const authenticationResult = await provider.authenticate(request, { authorization });
+      const authenticationResult = await provider.authenticate(request, { accessToken });
```

Comment:
This used to be `{ authorization }` instead of `{ accessToken }`, I assume this change was intentional?

Reply:
Yeah, token provider doesn't put `authorization` into session like `basic` does, it stores `accessToken` and `refreshToken` instead.